### DISCUSSION TITLE
How do you remove device groups

### DISCUSSION THREAD

**Discussion by Pshemas** [Ylianst/MeshCentral#6232] [Q&A]
I can't see a straightforward way to remove device groups - is there a way to do it via GUI?

The only mention in the docs that I see involves using meshctrl:

https://ylianst.github.io/MeshCentral/meshctrl/#creating-and-removing-device-groups

- is it the only way to achieve it?

**Comment by si458** [comment]
you can only delete a device group if the is no devices in the group
1. delete all devices from that group OR move them to a different group
2. click 'My Account'
3. click the group you want near the bottom
4. click the 'Delete Group' on the right hand side

edit: thanks for spotting tho in the new docs, the is no pic/steps saying HOW to delete the group!
